samples/streaming_sample/sample_stream.go

```diff
@@ -0,0 +1,208 @@
+	req = req.ModelMalwareScanTriggerReq(malwareScanReq)
+	res, _, err := malwareScanApi.StartMalwareScanExecute(req)
+	if err != nil {
+		log.Info().Msgf("Error in starting malware scan: %v", err)
+		return
+	}
+
+	respStr, err := json.Marshal(res)
+	if err != nil {
+		log.Info().Msgf("Failed to json marshal response, error:%v", err)
+		return
+	}
+
+	log.Info().Msgf("Successfully started Malware scan for deltas: %v", deltas)
+	log.Info().Msgf("StartMalwareScan Response: %s", respStr)
+
+}
+
+func startVulnerabilityScan(deltas []openapi.ModelNodeIdentifier) {
+	vulnerabilityScanApi := sampleApiClient.Client().VulnerabilityAPI
+	vulnerabilityScanReq := openapi.ModelVulnerabilityScanTriggerReq{}
+
+	vulnerabilityScanReq.Filters = *(openapi.NewModelScanFilterWithDefaults())
+	vulnerabilityScanReq.NodeIds = deltas
+	scanConfig := make([]openapi.ModelVulnerabilityScanConfigLanguage, 0)
+	scanConfig = append(scanConfig, openapi.ModelVulnerabilityScanConfigLanguage{"base"})
+	scanConfig = append(scanConfig, openapi.ModelVulnerabilityScanConfigLanguage{"java"})
+	scanConfig = append(scanConfig, openapi.ModelVulnerabilityScanConfigLanguage{"javascript"})
+	scanConfig = append(scanConfig, openapi.ModelVulnerabilityScanConfigLanguage{"golang"})
+	scanConfig = append(scanConfig, openapi.ModelVulnerabilityScanConfigLanguage{"ruby"})
+	scanConfig = append(scanConfig, openapi.ModelVulnerabilityScanConfigLanguage{"python"})
+	scanConfig = append(scanConfig, openapi.ModelVulnerabilityScanConfigLanguage{"php"})
+	scanConfig = append(scanConfig, openapi.ModelVulnerabilityScanConfigLanguage{"dotnet"})
+	scanConfig = append(scanConfig, openapi.ModelVulnerabilityScanConfigLanguage{"golang-binary"})
+	scanConfig = append(scanConfig, openapi.ModelVulnerabilityScanConfigLanguage{"rust-binary"})
+
+	vulnerabilityScanReq.ScanConfig = scanConfig
+
+	req := vulnerabilityScanApi.StartVulnerabilityScan(context.Background())
+	req = req.ModelVulnerabilityScanTriggerReq(vulnerabilityScanReq)
+	res, _, err := vulnerabilityScanApi.StartVulnerabilityScanExecute(req)
+	if err != nil {
+		log.Info().Msgf("Error in starting vulnerability scan: %v", err)
+		return
+	}
+
+	respStr, err := json.Marshal(res)
+	if err != nil {
+		log.Info().Msgf("Failed to json marshal response, error:%v", err)
+		return
+	}
+
+	log.Info().Msgf("Successfully started Vulnerability scan for deltas: %v", deltas)
+	log.Info().Msgf("StartVulnerabilityScan Response: %s", respStr)
+}
+
+func startComplianceScan(deltas []openapi.ModelNodeIdentifier) {
+	complianceScanApi := sampleApiClient.Client().ComplianceAPI
+	complianceScanReq := openapi.ModelComplianceScanTriggerReq{}
+
+	complianceScanReq.Filters = *(openapi.NewModelScanFilterWithDefaults())
+	complianceScanReq.NodeIds = deltas
+	req := complianceScanApi.StartComplianceScan(context.Background())
+	req = req.ModelComplianceScanTriggerReq(complianceScanReq)
+	res, _, err := complianceScanApi.StartComplianceScanExecute(req)
```

Comment:
Maybe we can create some utils to start scans - those utils would hide all the gory details around open API